src/language/rule/abstractRule.ts

```diff
@@ -17,38 +17,24 @@
 
 import * as ts from "typescript";
 
-import { getRuleSeverity, isRuleEnabled } from "../../configuration";
-import {arrayify} from "../../utils";
 import {doesIntersect} from "../utils";
 import {IWalker, WalkContext} from "../walker";
-import { IDisabledInterval, IOptions, IRule, IRuleMetadata, RuleFailure, RuleSeverity } from "./rule";
+import { IOptions, IRule, IRuleMetadata, RuleFailure, RuleSeverity } from "./rule";
 
 export abstract class AbstractRule implements IRule {
     public static metadata: IRuleMetadata;
     protected readonly ruleArguments: any[];
     protected readonly ruleSeverity: RuleSeverity;
+    public ruleName: string;
 
-    constructor(public readonly ruleName: string, private value: any, private disabledIntervals: IDisabledInterval[]) {
-        if (Array.isArray(value) && value.length > 1) {
-            this.ruleArguments = value.slice(1);
-        } else {
-            this.ruleArguments = [];
-        }
-
-        if (value.options) {
-            this.ruleArguments = arrayify(value.options);
-        }
-
-        this.ruleSeverity = getRuleSeverity(value);
+    constructor(private options: IOptions) {
```

Comment:
I don't think this should affect custom rules. Nobody should be calling the constructor except for ruleLoader in tslint itself.